# Lab 3 - EDA (Stats)
### Jackson Rolando

## Part 1: Review of Statistical Tests

My initial hypothesis:
The GPAs of students who play video games regularly are significantly lower than those who do not.

My answers to the questions:
1. A two-sample t-test is be used to compare a measured continuous variable between two groups. 
2. This situation is perfect for a two-sampled t-test. There are two groups, gamers and non-gamers, and we have the continuous variable of a GPA. 
Are there any particular assumptions that the t-test makes that may not hold here?
3. We don't know how our distributions fall. The test assumes a normal distribution, which we have no idea of confirming. The standard deviations are similar, but since they are close, and we have more than 30 samples in each category, the mismatch shouldn't be an issue.

**Null Hypothesesis:** There is not a significant difference between the groups.

**Alernative Hypothesesis:** There is a significant difference between the groups, and the gamers have a higher GPA. 

Here we apply the two-sampled t-test on the sample data: 
> 68 students said they play video games regularly, while 32 students said they did not. The 68 games have an average GPA of 3.4 with a standard deviation of 1.2, while the 32 non-gamers have an average GPA of 3.3 with a standard deviation of 1.1.

In [2]:
import scipy.stats as stats
stats.ttest_ind_from_stats(3.4, 1.2, 68, 3.3, 1.1, 32)

Ttest_indResult(statistic=0.39893881176878243, pvalue=0.6908062583072547)

The test yielded a p-value of 0.6908, way higher than the required maximum of 0.01. This means that we cannot reject the null hypothesis, as there is not a significant enough difference between the two groups of students.

The result of the t-test cannot prove my initial hypothesis either. My mentioning of one group having 'significantly lower' GPAs was just another way of saying there was a significant negative difference, which the t-test disproved. 

## Part 2: Exploring Additional Statistical Tests
a.  List  the  two  types  of  variables  for  which  the  test  is  appropriate.    Indicate  any 
assumptions that you would need to be aware of. 
 
b. Write down the general forms of the null and alternative hypotheses (one sentence per 
hypothesis). 
 
c. In your own words, write what it would mean if the test did and did not indicate 
statistical significance.

### Correlation and Linear Regression

A test for linear correlation is best done between two continuous features measured from the same sample. 

### Kruskal-Wallis Test

### Chi Squared Test